joomla/joomla-cms
# [4.3] Codemirror wrong path

Pull Request for Issue #40049 .

### Summary of Changes



### Testing Instructions

- make codemirror the default browser
- Enable keymaping
- Check that codemirro and keybinding work


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:
